packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -128,7 +128,13 @@ export async function process(options: ProcessBundleOptions): Promise<ProcessBun
       babelrc: false,
       // modules aren't needed since the bundles use webpack's custom module loading
       // 'transform-typeof-symbol' generates slower code
-      presets: [['@babel/preset-env', { modules: false, exclude: ['transform-typeof-symbol'] }]],
+      presets: [[
+        require.resolve('@babel/preset-env'),
+        {
+          modules: false,
```

Comment:
Can you move the relevant comments next to each option as well?

Reply:
Moved